messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ActiveMQServerControlWriteHandler.java

```diff
@@ -70,6 +72,21 @@ protected boolean applyUpdateToRuntime(final OperationContext context, final Mod
                                            final ModelNode newValue, final ModelNode currentValue,
                                            final HandbackHolder<Void> handbackHolder) throws OperationFailedException {
         AttributeDefinition attr = getAttributeDefinition(attributeName);
+        if(attr == ServerDefinition.JOURNAL_FILE_SIZE){
```

Comment:
As I commented on WFLY-6268, It should only validate against multiple value of 512 for ASYNCIO journal. for NIO journal there is no such alignment restriction.